openhands/resolver/send_pull_request.py

```diff
@@ -334,6 +334,23 @@ def reply_to_comment(github_token: str, comment_id: str, reply: str):
     response.raise_for_status()
 
 
+def send_comment_msg(base_url: str, issue_number: int, github_token: str, msg: str):
```

Comment:
Just a nit: could we add a docstring?

Reply:
@openhands-agent can you add a docstring to this new function? Note how the other docstrings are in the file, and keep the same style!

In particular, add the args without types.

Reply:
@malhotra5 this is funny, it added a docstring to another function 😂 https://github.com/All-Hands-AI/OpenHands/pull/5351/commits/1e26cf0d666a4504b0c7a1cab3793d13b31f3f3a

Reply:
@openhands-agent good job, now add a docstring to the new send_comment_msg function. Similarly, look how other docstrings are in the file, and keep the same style!